### ISSUE TITLE
Issue in train

### ISSUE THREAD

**Issue by hyz3203** [iperov/DeepFaceLab#663]
## Expected behavior

*i'm following Basic workflow video,but in the train it don't work*

## Actual behavior

*Model first run.
Enable autobackup? (y/n ?:help skip:n) : y
Write preview history? (y/n ?:help skip:n) : y
Choose image for the preview history? (y/n skip:n) : y
Target iteration (skip:unlimited/default) :
0
Batch_size (?:help skip:0) : 8
Flip faces randomly? (y/n ?:help skip:y) :
y
Use lightweight autoencoder? (y/n, ?:help skip:n) : y
Use pixel loss? (y/n, ?:help skip: n/default ) :
n
Using plaidml.keras.backend backend.
INFO:plaidml:Opening device "opencl_amd_ellesmere.0"
Loading: 100%|####################################################################| 7011/7011 [00:09<00:00, 732.12it/s]
*
Then, Program stuck.

## Steps to reproduce

The data_dst and data_src extract faces  work.I try to convert H64 without train,it works.
Is it the cause of the GPU?

##Other relevant information
my rig:
Asus Motar B360
AMD 2600
sapphire rx580 2048sp 8g
win 10

I`m sorry for my poor english

**Comment by iperov** [comment]
dfl 1.0 and amd cards are no longer supported.
